lls/nd.c

```diff
@@ -389,7 +392,7 @@ process_nd_neigh_solicitation(struct lls_config *lls_conf, struct rte_mbuf *buf,
 			ndopts.opt_array[ND_OPT_SOURCE_LL_ADDR];
 
 		/* Update resolution of source of Solicitation. */
-		rte_memcpy(mod_req.ip_be, ipv6_hdr->src_addr,
+		rte_memcpy(mod_req.addr.ip.v6.s6_addr, ipv6_hdr->src_addr,
 			lls_conf->nd_cache.key_len);
```

Comment:
1. Shouldn't we replace `lls_conf->nd_cache.key_len` with `sizeof(mod_req.addr.ip.v6.s6_addr)` here?

2. Is field `key_len` still needed at all?